package.json

```diff
@@ -1,37 +1,53 @@
 {
   "name": "resub",
-  "version": "1.0.0",
+  "version": "1.0.1",
   "description": "A library for writing React components that automatically manage subscriptions to data sources simply by accessing them.",
   "author": "David de Regt <David.de.Regt@microsoft.com>",
   "scripts": {
-    "prepublish": "tsc",
-    "webtestbuild": "webpack",
-    "build": "npm run tslint && tsc",
-    "tslint": "tslint --project tsconfig.json -r tslint.json -r ./node_modules/tslint-microsoft-contrib --fix || true"
+    "prepare": "npm run build",
+    "clean": "rimraf dist/",
```

Comment:
i think you'll need rimraf as a devdependency for this to work, right?

Reply:
Yes, you are right, I've added.